cloudformation_seed/cfn_stackset.py

```diff
@@ -496,6 +497,22 @@ def rollout_stackset(self):
         else:
             self.rollout_accounts()
 
+    def check_stack_instances(self):
+        failures = []
+        c = util.session.client('cloudformation')
+        for page in c.get_paginator('list_stack_instances').paginate():
+            for summary in page['Summaries']:
+                if summary['Status'] in ['OUTDATED', 'INOPERABLE']:
```

Comment:
I think as soon as a StackSet template is updated, they all become Outdated immediately. So this may require a wait of some sort.